src/components/views/rooms/MessageComposer.js

```diff
@@ -437,6 +437,7 @@ export default class MessageComposer extends React.Component {
                     const canEndConf = WidgetUtils.canUserModifyWidgets(this.props.room.roomId);
                     controls.push(
                         <HangupButton
+                            key="controls_hangup"
```

Comment:
This is okay, but is it mean to be in this PR...?

Reply:
I did intentionally put it in here, sure its not release-blockery but
> Resolve couple of React warnings/errors